### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by MarSYs1** [witekbobrowski/EPUBKit#11] [General]
Does ePUBKit not have documentation to assist in implementation? 
As for building views, can I use SwiftUI to display the functionalities?

**Comment by witekbobrowski** [comment]
Hello @MarSYs1 

The documentation is not really there, but since the library is super small, with the help of Readme and going thought the code you can quickly learn all the essential information on how to use it :)

When it comes to building UI, you can use any framework you want: UIKit, AppKit, and of course SwiftUI. 

Cheers!

**Reply by MarSYs1** [reply]
Hello @witekbobrowski !!

Thank you for your attention. I really want to use it. I will explore it. If, by chance, the documentation is ready, don't forget to mention it to me. For now, I will head towards the bullets.
